patches/node-version.diff

```diff
@@ -87,21 +87,3 @@ Index: code-server/lib/vscode/build/lib/util.ts
  	return result;
  }
 -
-Index: code-server/lib/vscode/remote/.yarnrc
-===================================================================
---- code-server.orig/lib/vscode/remote/.yarnrc
-+++ /dev/null
-@@ -1,4 +0,0 @@
--disturl "http://nodejs.org/dist"
--target "14.16.0"
--runtime "node"
--build_from_source "true"
-Index: code-server/lib/vscode/.yarnrc
-===================================================================
---- code-server.orig/lib/vscode/.yarnrc
-+++ /dev/null
-@@ -1,4 +0,0 @@
--disturl "https://electronjs.org/headers"
--target "13.5.2"
--runtime "electron"
--build_from_source "true"
```

Comment:
I wasn't sure how to read this diff. It seemed like we didn't have those files and the diff shows what was in them, right? Not sure if that would fix the node-version issues. I wish we could remove this patch.

Reply:
This one always gets conflicts because they update the version and we delete the file.

Maybe instead of patching we delete them as part of a `yarn` pre-install?

Reply:
Ooo, that's a great idea! 

> postinstall is called after a package's dependency tree changes are written to the disk -- e.g. after a dependency or transitive dependency is added, removed, or changed. It is guaranteed to be called in topological order (in other words, your dependencies' postinstalls will always run before yours). **For backwards compatibility, preinstall and install are called as part of postinstall**.

Hmm...just tested. Looks like preinstall and postinstall are run everytime you run `yarn` or `yarn add` 🤔

<img width="947" alt="image" src="https://user-images.githubusercontent.com/3806031/165814511-890509ee-9961-4961-b6d4-fb2682ec4b17.png">

I guess that doesn't matter but we'd have to add a check that deletes the file if exists I guess?